BatchManagement.json

```diff
@@ -0,0 +1,1618 @@
+          "description": "The type of account key to regenerate.",
+          "enum": [
+            "Primary",
+            "Secondary"
+          ],
+          "x-ms-enum": {
+            "name": "AccountKeyType",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "keyName"
+      ],
+      "description": "Parameters supplied to the RegenerateKey operation."
+    },
+    "BatchAccountKeys": {
+      "properties": {
+        "accountName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The Batch account name."
+        },
+        "primary": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The primary key associated with the account."
+        },
+        "secondary": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The secondary key associated with the account."
+        }
+      },
+      "description": "A set of Azure Batch account keys."
+    },
+    "ActivateApplicationPackageParameters": {
+      "properties": {
+        "format": {
+          "type": "string",
+          "description": "The format of the application package binary file."
+        }
+      },
+      "required": [
+        "format"
+      ],
+      "description": "Parameters for an activating an application package."
+    },
+    "ApplicationCreateParameters": {
+      "properties": {
+        "allowUpdates": {
+          "type": "boolean",
+          "description": "A value indicating whether packages within the application may be overwritten using the same version string."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name for the application."
+        }
+      },
+      "description": "Parameters for adding an Application."
+    },
+    "Application": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "A string that uniquely identifies the application within the account."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name for the application."
+        },
+        "packages": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationPackage"
+          },
+          "description": "The list of packages under this application."
+        },
+        "allowUpdates": {
+          "type": "boolean",
+          "description": "A value indicating whether packages within the application may be overwritten using the same version string."
+        },
+        "defaultVersion": {
+          "type": "string",
+          "description": "The package to use if a client requests the application but does not specify a version."
+        }
+      },
+      "description": "Contains information about an application in a Batch account."
+    },
+    "ApplicationPackage": {
+      "properties": {
+        "id": {
```

Comment:
This type doesn't match the ARM proxy specification and will be reworked in a future API version.